django_mongodb_backend/forms/fields/array.py

```
@@ -24,6 +26,9 @@ def __init__(self, base_field, *, delimiter=",", max_length=None, min_length=Non
         if max_length is not None:
             self.max_length = max_length
             self.validators.append(ArrayMaxLengthValidator(int(max_length)))
+        if size is not None:
```

While I don't see a precedent in existing form fields, I suppose we should raise an exception for mutual exclusivity between size & min/max_length similar to what you currently done in the model field. (Forms don't have system checks since unlike models, they can't be collected at application start up time.)

---

is another validator needed for that?

---

No, I meant a programmer facing error if both params are passed. `ImproperlyConfigured` seems appropriate.